app/views/library/dashboard/components/SidebarComponent.vue

```diff
@@ -68,6 +68,27 @@ export default {
 <style scoped lang="scss">
 @import "../css-mixins/variables";
 
+@media print {
+  .sidebar {
+    width: 1024px !important;
```

Comment:
do we need this fixed value?

Reply:
yes. this value is good for a4 page pdf. if not, on different screen the pdf may shows different